src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -97,10 +97,14 @@ private void ExecuteSearchElement(ListBoxItem listBoxItem)
                 if (searchElement.CreateAndConnectCommand.CanExecute(port.PortModel))
                 {
                     searchElement.CreateAndConnectCommand.Execute(port.PortModel);
+                    var searchElementInfo = ViewModel.IsDisplayingMLRecommendation ?
+                        searchElement.FullName + " " + port.PortModel.Index.ToString() + " " + port.PortName + " " + port.NodeViewModel.OriginalName + " " +  ViewModel.ServiceVersion
```

Comment:
For the ML mode, the component name will be a long string with a combination of information @varvaratou FYI

Reply:
I think the threshold at this point may or may not affect what option the user picks so we want to rule it out as a factor. The aim of the feature is that the user was given the most accurate results with high confidence. Here currently, it is assuming the user was given those results, and see what they pick.